packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @mongodb-js/compass/no-leafygreen-outside-compass-components */
```

Comment:
I don't see eslint failing with this error locally if I remove this, do we need it?

Reply:
I couldn't get it to work consistently no matter what I did and I feel like the comment is harmless.

Reply:
I think it's confusing to have a rule that only allows to use something in compass-components to be disabled in compass-components, can we at least disable it for the whole workspace by extending the eslint config and not in a few random files? Let's also open a ticket to remove this rule if it's in the way. If we just work around our eslint config we will end up in the same state that the previous config was

Reply:
Even after re-bootstrapping and restarting all the relevant plugins and servers in vscode if I remove this line now I immediately see all the errors again in vscode.

Reply:
Nevermind. Worked third try. I removed the lines.